frontend/src/components/Utils/temp-auth-utils.js

```diff
@@ -16,15 +16,15 @@ import authUtils from '../AuthUtils';
  * @return {string} Fake user uid string the form '_userEmail_'.
  */
 export function getCurUserUid() {
-  return getUserUidArrFromUserEmailArr([authUtils.getCurUserEmail()]).pop();
+  return '_' + authUtils.getCurUserEmail() + '_';
```

Comment:
This is fine since it's temporary. Normally if you have duplicate logic in multiple places, it makes more sense to have a common function (like `addUnderscoresToString` or something) and then have both getCurUserUid and GetUserUidArrFromUserEmailArr call that common function. Then you avoid calling a map function on a single element and also avoid having duplicate underscore logic. Again, since this is a simple example that is expected to go away, it's fine as-is.

Reply:
Yeah I agree, that is why I originally called `getUserUidArrayFromUserEmailArr()` from `getCurUserUid` but I forgot to make them a common function like `addUnderscoresToString` when making the simplification to `getCurUserUid`. I will make that change even though this is temporary.